tests/Checkbox.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Checkbox } from "../lib/components";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("Checkbox", () => {
+  it("should render without error", () => {
+    const { getByText } = render(<Checkbox label="Checkbox" />);
+    expect(getByText("Checkbox")).toBeInTheDocument();
+  });
```

Comment:
Let's do `getByRole("checkbox")` or `getByLabelText("Checkbox")` instead of `getByText` here.